apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -32,12 +39,19 @@ const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
                     user: { connect: { id: context.authedItem.id } },
                 }
                 const [existingResident] = await ResidentAPI.getAll(context, {
-                    address,
+                    address_i: address,
                     unitName,
                     user: { id: context.authedItem.id },
+                }, {
+                    first: 1,
                 })
+                const propertyAddress = removeFlatNum(address, addressMeta.data.flat_type)
```

Comment:
I've created a function `getAddressUpToBuildingFrom` with the same intent some time before.
Can it be reused in your case?

Reply:
done